src/sider/arbeidsuforhet/avslag/ArbeidsuforhetAvslagSide.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ReactElement } from "react";
+import Side from "@/sider/Side";
+import Sidetopp from "@/components/Sidetopp";
+import SideLaster from "@/components/SideLaster";
+import UtdragFraSykefravaeret from "@/components/utdragFraSykefravaeret/UtdragFraSykefravaeret";
+import * as Tredelt from "@/sider/TredeltSide";
+import { Menypunkter } from "@/components/globalnavigasjon/GlobalNavigasjon";
+import { VurderingHistorikk } from "@/sider/arbeidsuforhet/historikk/VurderingHistorikk";
+import { useArbeidsuforhetVurderingQuery } from "@/data/arbeidsuforhet/arbeidsuforhetQueryHooks";
+import { ArbeidsuforhetAvslag } from "@/sider/arbeidsuforhet/avslag/ArbeidsuforhetAvslag";
+
+const texts = {
+  title: "Arbeidsuførhet",
+};
+
+export const ArbeidsuforhetAvslagSide = (): ReactElement => {
+  const { isLoading, isError } = useArbeidsuforhetVurderingQuery();
+
+  return (
+    <Side tittel={texts.title} aktivtMenypunkt={Menypunkter.ARBEIDSUFORHET}>
+      <Sidetopp tittel={texts.title} />
+      <SideLaster henter={isLoading} hentingFeilet={isError}>
+        <Tredelt.Container>
+          <Tredelt.FirstColumn>
+            <ArbeidsuforhetAvslag />
+            <VurderingHistorikk />
```

Comment:
Vi ender opp med flere sånne sider fordi det er ulike routes, sant? Bare tenkte på at det var kjipt å gjøre det samme med tredeling flere steder for arbeidsuførhet, men det er kanskje ikke noe man kan komme seg utenom 🤔

Reply:
Det er sikkert mulig å gjøre det annerledes, men jeg vet ikke hva som er best 😞